app/models/concerns/bib_record.rb

```diff
@@ -62,12 +62,10 @@ def call
 
     # Use Redis to store this globally once an hour
     def item_type_mapping
-      redis = Redis.new
-      return JSON.parse(redis.get('item_type_map')) unless redis.get('item_type_map').nil?
-
-      client = SymphonyClient.new
-      item_type_map = client.get_item_type_map.to_json
-      redis.setex('item_type_map', 3600, item_type_map)
-      JSON.parse(item_type_map)
+      Rails.cache.fetch(['item_type_map', __method__], expires_in: 60.minutes) do
```

Comment:
Please note I changed this to a cache option (which is still Redis but it is more semantically correct and would still work if we swapped the cache backend)

Reply:
makes sense!